cmake_modules/parquet-cppConfig.cmake.in

```diff
@@ -0,0 +1,15 @@
+set(PARQUET-CPP_VERSION x.y.z)
```

Comment:
is this file path the typical style (i.e. would it be obvious to other CMake users what is in here)?

Reply:
Yes, see https://cmake.org/cmake/help/v3.5/module/CMakePackageConfigHelpers.html?highlight=cmakepackageconfighelpers
This allows find_package to be used in config mode i.e.,
`find_package(parquet-cpp 1.3.0 REQUIRED CONFIG)`